desktop/src/@batch-flask/ui/list-and-show-layout/entity-details-list.html

```diff
@@ -7,6 +7,8 @@
         <bl-clickable *ngIf="enableAdvancedFilter"
             class="toggle-advanced-filter"
             title="{{'entity-details-list.toggleAdvancedFilter' | i18n}}"
+            matTooltip="Toggle advanced filter"
```

Comment:
I removed the `matTooltip` attribute, using the `title` attribute instead, for a very similar issue, AB#487. I think your change makes better sense, and it preserves the Material library's tooltip. Can you just remove the `title` attribute but also use the localized version of the string (the contents of the attribute value)?

Reply:
Will do! I'd also suggest using `matTooltip` for cases like this because it has a slight adverting over `title` as it makes tooltips visible on keyboard focus of interactive items, whereas `title` tooltips are only visible on mouse hover. `matTooltip` also adds a couple other `aria` attributes by default.